src/Maker/MakeSerializerNormalizer.php

```diff
@@ -35,25 +39,38 @@ public static function getCommandDescription(): string
         return 'Creates a new serializer normalizer class';
     }
 
-    public function configureCommand(Command $command, InputConfiguration $inputConf)
+    public function configureCommand(Command $command, InputConfiguration $inputConfig): void
     {
         $command
             ->addArgument('name', InputArgument::OPTIONAL, 'Choose a class name for your normalizer (e.g. <fg=yellow>UserNormalizer</>)')
             ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeSerializerNormalizer.txt'))
         ;
     }
 
-    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator)
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator): void
     {
         $normalizerClassNameDetails = $generator->createClassNameDetails(
             $input->getArgument('name'),
             'Serializer\\Normalizer\\',
             'Normalizer'
         );
 
+        $useStatements = new UseStatementGenerator([
+            NormalizerInterface::class,
+            ObjectNormalizer::class,
+        ]);
+
+        if ($cacheable = interface_exists(CacheableSupportsMethodInterface::class)) {
```

Comment:
This will always exist, right? This interface exists at least as far back as 4.4.

Reply:
good catch